sips/pending/_posts/2016-01-11-static-members.md

```diff
@@ -132,12 +134,82 @@ Java allows classes to define static methods with the same name and signature as
+since fields are initialized in the order `as written`, similar to how normal fields are initialized.
+
+The `@static` proposal is similar to `@tailrec` in a sense that it fails compilation in the case where the user did not write code that follows the aforementioned rules. 
+These rules exist to enforce the unlikelyhood of an observable difference in semantics if `@static` annotations are dropped;
+The restrictions in this SIP make it hard to observe changes in initialization within the same object.
+It is still possible to observe those changes using multiple classes and side effects within initializers:
+
+```scala
+{% highlight scala %}
+class C { 
+  val x = {println("x"); 1 } 
+}
+
+
+object O extends C { 
+  val y = { println("y"); 2 } 
+  // prints:
+  // x
+  // y
+}
+    
+object Os extends C { 
+  @static val y = { println("y"); 2 }
+   // prints:
+   // y
+   // x
+}
+{% endhighlight %}
+```
+
+Static fields can be initialized earlier than they used to be initialized while being non-static, but never later.
+By requiring `@static` first to be defined first inside the object, 
+we guarantee that you can't observe the changes in initialization withing the same object without resorting to code which either uses `Unsafe` or exhibits undefined behaviour under the JVM.
+
+## Could `@static` be a `@tailrec`-like annotation that doesn't affect code generation but only checks ##
+Unlike `@tailrec` this annotation does affect the binary API and dropping such an annotation would be a binary incompatible change. This is why authors believe that developers should be in full control of what is static.
+
+## Alternative: Emitting fields of objects as static by default ##
+An alternative to this proposal would be to emit all the fields defined in objects as static.
+Unfortunately this gets us under dark waters when we try to figure out in the following example:
+
+```
+{% highlight scala %}
+class Super {
+ val c = {println(1); 1}
+}
+object Object extends Super {
+ override val c = {println(2); 2}
+ val d = {println(3); 2}
+}
+{% endhighlight %}
+```
+Let's consider possible options:
+ - if the field `c` is emitted as `static` on the bytecode level, it will be initialized before the `c` in superclass is initialized, reordering side-effects in initializers;
+ - if the field `c` is emitted as `static` on the bytecode level it will use a different storage that the memory layout defined by `Super`, leading to potential bugs and wasted memory;
+ - if the field `c` is _not_ emitted as `static` but the field `d` is, then the order of initialization would also be affected, reordering side-effects.
+ 
+Based on the previous study done in preparation for this SIP, the authors believe that the only reasonable way to maintain current sematics would be to say that such alternative would require these rules:
+ - only the fields which were not declared by parents of the object can be emitted as static;
```

Comment:
I think you need an space here for Jekyll to show it as an enumeration, the same for line 203 of this diff, in "let's consider possible options".